src/common/ipfs.ts

```diff
@@ -67,6 +67,29 @@ export async function uploadMetadata(
   if (typeof metadata === "string") {
     return metadata;
   }
+  async function _fileHandler(object: any) {
+    const keys = Object.keys(object);
+    for (const key in keys) {
+      if (
+        Buffer.isBuffer(object[keys[key]]) ||
```

Comment:
This condition is really obscure. We just need to verify if the property is a `File` or `Buffer`.

I'm pushing a change for this.